recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,5 @@
+cudnn:                                    # [linux64]
```

Comment:
yep... wouldn't have guessed that... also the fact there's the strings `None` and `undefined` within spitting distance of each other terrifies me...

Reply:
Yeah the pinnings are fun... 😄

I found it by comparing `recipe/conda-build-config.yaml` with the global pinnings (more specifically, [here](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/d8507e01df35678a41ad432e9dd3278f73a689e0/recipe/conda_build_config.yaml#L146-L150)).

I've made a PR to the bot to try and improve the error message a little (https://github.com/conda-forge/webservices-dispatch-action/pull/9). If there is anything that you think would help make it clearer let me know.

Reply:
Will comment over there!